platform/commonUI/edit/test/actions/SaveAsActionSpec.js

```diff
@@ -156,6 +156,28 @@ define(
                 expect(SaveAsAction.appliesTo(actionContext)).toBe(false);
             });
 
+            it("uses the editor capability to save the object", function () {
+                mockEditorCapability.save.andReturn(new Promise(function () {}));
```

Comment:
Ah alright, that definitely confused me then. Pushed the latest commit w/ the `CreationService` removed. Ran some final tests and everything seems to work just fine :)

Let me know if anything else needs changing.